{"uuid": "65798c36-32d3-4979-91a1-0a0bb03cc6c8", "children": ["be63a7af-9da7-43c4-9d35-44395a351949"], "befores": [{"name": "create_comment", "status": "broken", "statusDetails": {"message": "KeyError: 'id'\n", "trace": "  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 166, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/pluggy/_callers.py\", line 102, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/venv/lib/python3.12/site-packages/_pytest/fixtures.py\", line 910, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/home/berserker/python/api_automation_2024/todo_api/conftest.py\", line 115, in create_comment\n    id_comment_created = response[\"body\"][\"id\"]\n                         ~~~~~~~~~~~~~~~~^^^^^^\n"}, "start": 1712609580523, "stop": 1712609581016}], "start": 1712609580523, "stop": 1712609581030}